packages/plugins/experimental/plugin-excalidraw/src/hooks/useStoreAdapter.ts

```diff
@@ -28,7 +28,7 @@ export const useStoreAdapter = (
     }
 
     const t = setTimeout(async () => {
-      await adapter.open(createDocAccessor(object, ['content']));
+      await adapter.open(createDocAccessor(object, ['canvas', 'content']));
```

Comment:
```suggestion
      await adapter.open(createDocAccessor(object.canvas, ['content']));
```
is there a difference between these two? if the canvas isn't loaded will creating a deeply nested doc accessor load the object? shouldn't merge this without the corresponding migration?

Reply:
ok, i cleaned this up before this commit -- and i think i had confused myself. no, this is correct. and a migration won't be needed. but it doesn't work in the storybook.